frontend/src/components/Shared/ForgetPassword/forgetpassword.jsx

```
@@ -97,109 +160,83 @@
+            />
           </form>
 
-          <p className="mt-5 sm:mt-6 text-center text-sm text-gray-400">
+          {/* Back to login link */}
+          <p className="mt-4 sm:mt-5 md:mt-6 text-center text-xs sm:text-sm text-gray-400 sticky bottom-0 py-2 ">
             <Link
               to="/login"
-              className="font-medium text-blue-400 hover:text-blue-300"
+              className="font-medium text-blue-400 hover:text-blue-300 focus:outline-none focus:underline inline-flex items-center"
             >
+              <svg xmlns="http://www.w3.org/2000/svg" className="h-3.5 w-3.5 mr-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)